### PR TITLE
[3596][15.0][MIG] purchase_invoice_indicator: Migration to 15.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[3596](https://www.quartile.co/web#view_type=form&model=project.task&id=3596&active_id=3596&menu_id=)

### CODE REVIEW

File: purchase_invoice_indicator/models/account_move.py [hunk 1]
```diff
@@ -4,12 +4,12 @@
 from odoo import api, fields, models
 
 
-class AccountInvoice(models.Model):
-    _inherit = "account.invoice"
+class AccountMove(models.Model):
+    _inherit = "account.move"
 
-    is_invoice = fields.Boolean("Is Invoice", readonly=True, store=True)
+    is_invoice_issuer = fields.Boolean(string="Is Invoice", readonly=True, store=True)
```
Comment (kanda999) [inline purchase_invoice_indicator/models/account_move.py]: Why you change field name?

Comment (AungKoKoLin1997) [reply purchase_invoice_indicator/models/account_move.py]: In odoo15, there is `is_invoice` method. So, if we use is_invoice field and call this function, odoo call field and get error.https://github.com/odoo/odoo/blob/2c6d0d7526ca75ec78d92e10e73b53330a42cb4f/addons/account/models/account_move.py#L544

File: purchase_invoice_indicator/models/account_move.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    is_invoice_issuer = fields.Boolean(string="Is Invoice", readonly=True, store=True)
```
Comment (yostashiro) [inline purchase_invoice_indicator/models/account_move.py]:
```suggestion
    is_invoice_issuer = fields.Boolean()
```

We should do the same in the 11.0 module (with migration scripts).

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Review by kanda999 — APPROVED** [review]
Functional review

**Pull request merged**